Competition2018/src/main/java/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -128,110 +128,60 @@ public void setupDriveCommands(OperatorInterface oi, ResetDistanceCommand resetD
 
         oi.driverGamepad.getPovIfAvailable(0).whenPressed(velocity);
         oi.driverGamepad.getPovIfAvailable(180).whenPressed(arcade);
-        
```

Comment:
Can we undo the formatting in this file given there weren't any code changes?

Reply:
yes, but I don't know how to.